library/systemd/src/lib/yast2/service_widget.rb

```diff
@@ -60,15 +65,32 @@ module Yast2
   # @todo Allow to specify the widget ID. Currently, it uses always the same, so you can not use
   #   more than one instance of this class at the same time.
   class ServiceWidget
+    extend Yast::I18n
     include Yast::I18n
     include Yast::Logger
     include Yast::UIShortcuts
 
+    # It stores the selected service action
+    #
+    # @return [Symbol] service action (:nothing for no action)
+    # @see Yast2::SystemService#action
+    attr_reader :current_action
+
     # creates new widget instance for given service
     # @param service [Yast2::SystemService,Yast2::CompoundService] service
     def initialize(service)
       textdomain "base"
       @service = service
+      # When the service is active, by default, propose to reload or restart
+      # it after writting the configuration (bsc#1158946)
+      init_default_action
+    end
+
+    # Set the given action as the current action selected by the widget
+    #
+    # @param value [Symbol, nil]
```

Comment:
please document what means nil here. As it can be also interpreted as reset to default.